src/Models/ApplicantsModel.php

```diff
@@ -97,6 +105,50 @@ public function getById(int $id): Applicant|false
         return ApplicantHydrator::hydrateSingle($data, $applicationEntity);
     }
 
+    public function editApplicant($details){
+        $query1 = $this->db->prepare("UPDATE `applicants` SET `name` = :name, `email` = :email WHERE `id` = :id;");
+        $query1->execute([
+            'id' => $details['id'],
+            'name' => $details['name'],
+            'email' => $details['email']
+        ]);
+
+        $query2 = $this->db->prepare("UPDATE `applications`
+        SET `why` = :why,
+            `experience` = :experience,
+            `diversitech` = :diversitech,
+            `circumstance_id` = :circumstance_id,
+            `funding_id` = :funding_id,
+            `cohort_id` = :cohort_id,
+            `dob` = :dob,
+            `phone` = :phone,
+            `address` = :address,
+            `heard_about_id` = :heard_about_id,
+            `age_confirmation` = :age_confirmation,
+            `newsletter` = :newsletter,
+            `eligible` = :eligible,
+            `terms` = :terms
+            WHERE `applicant_id` = :id");
+
+        $query2->execute([
+            'id' => $details['id'],
+            'why' => $details['why'],
+            'experience' => $details['experience'],
+            'diversitech' => $details['diversitech'],
+            'circumstance_id' => $details['circumstance_id'],
+            'funding_id' => $details['funding_id'],
+            'cohort_id' => $details['cohort_id'],
+            'dob' => $details['dob'],
+            'phone' => $details['phone'],
+            'address' => $details['address'],
+            'heard_about_id' => $details['heard_about_id'],
+            'age_confirmation' => $details['age_confirmation'],
+            'newsletter' => $details['newsletter'],
+            'eligible' => $details['eligible'],
+            'terms' => $details['terms']
+        ]);
```

Comment:
This changes an Application in the database, so should be in a function in the application model.

It's going to be very similar to the controller for the add applicant page.

Reply:
done